bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -447,9 +447,6 @@ spec:
             - name: empty-dir
               mountPath: /opt/bitnami/postgresql/tmp
               subPath: app-tmp-dir
-            - name: empty-dir
-              mountPath: /opt/bitnami/postgresql/logs
-              subPath: app-logs-dir
```

Comment:
Is there any option in the chart to switch from throwing logs to stdout/stderr to save them in a file?

Reply:
No that I know of. If a user changes the log file path via custom configuration, then they can always make use of extra volume mounts